app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    command = command.split()
```

Comment:
add additional checks to make sure that the  command  consists of three parts and that it starts with "cp"